package.json

```diff
@@ -87,7 +87,7 @@
     "inject-css": "^0.1.1",
     "jazzicon": "^1.2.0",
     "loglevel": "^1.4.1",
-    "menu-droppo": "^1.1.0",
+    "menu-droppo": "2.0.0",
```

Comment:
Might be low priority: could this be `2.0.1` since it's merged in `menu-droppo`?